File: javascript/media_pares.js
const n = parseInt(prompt("Quantos elementos vai ter o vetor?"));

let vetor = [], mediaPares = 0, qtdPares = 0;

for(i = 0; i < n; i++) {
    vetor[i] = parseFloat(prompt("Digite um numero:"));

    if (vetor[i] % 2 == 0) {
        mediaPares += vetor[i];
        qtdPares++;
    }
}

mediaPares /= qtdPares;

if (qtdPares == 0) {
    console.log("NENHUM NUMERO PAR");
} else {
    console.log("MEDIA DOS PARES = %.1f", mediaPares);
}

File: python/media_pares.py
n = int(input("Quantos elementos vai ter o vetor? "))

vetor = []
soma_pares = 0
qtd_pares = 0

for i in range(0, n):
    vetor.append(int(input("Digite um numero: ")))

    if vetor[i] % 2 == 0:
        soma_pares += vetor[i]
        qtd_pares += 1

if qtd_pares == 0:
    print("NENHUM NUMERO PAR")
else:
    media_pares = soma_pares / qtd_pares

    print(f"MEDIA DOS PARES = {media_pares:.1f}")
